modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -452,6 +468,35 @@ public int getOneToManyRelatedObjectEntriesCount(
 		return objectEntryPersistence.dslQueryCount(dslQuery);
 	}
 
+	@Override
+	public List<Map<String, Object>> getOneToManySystemRelatedObjectEntries(
```

Comment:
Could we rename to getOneToManyRelatedSystemObjectEntries ? I think that the "System" should be near to "ObjectEntries" ?